frontend/components/card/search-result/CardSearchResult.vue

```diff
@@ -35,13 +41,16 @@ import type { Event } from "../../../types/event";
 import type { Organization } from "../../../types/organization";
 import type { Resource } from "../../../types/resource";
 import type { User } from "../../../types/user";
+import type { Discussion } from "../../../types/discussion";
+import CardDiscussionEntry from "./CardDiscussionEntry.vue";
```

Comment:
A quick note here that as with the other issue we do not need to import components because Nuxt imports them automatically, but we do need the type :) Obviously no worries as these reviews are coming so close to one another!